atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/schema/KeyValueServiceMigrator.java

```diff
@@ -206,6 +206,8 @@ private void migrateTables(Set<TableReference> tables,
                                long migrationTimestamp,
                                ExecutorService executor,
                                GeneralTaskCheckpointer checkpointer) {
+        processMessage("Migrating tables at migrationTimestamp " + migrationTimestamp,
```

Comment:
really wish this was named something better than `processMessage`.. is it logging this? anyway, not a blocker for this PR